src/theme/colors.ts

```diff
@@ -3,9 +3,9 @@ import {StatusBarProps, TextStyle} from 'react-native';
 
 const backgrounds = {
   light__level0: '#FFFFFF',
-  light__level1: '#F5F5F6',
-  light__level2: '#EBECED',
-  light__level3: '#E1E3E4',
+  light__level1: '#E7E8E9',
```

Comment:
Ah gray_50 I see

Reply:
Ah, haha, I "solved" the problem :flushed: :joy:

Reply:
I'll revert this one :point_up_2: Actually not intentional, I think I may have misinterpreted this color as being the "new" gray_100 value at the time the change happened..